pkg/kubelet/container/runtime.go

```diff
@@ -262,6 +263,8 @@ type PodStatus struct {
 	Namespace string
 	// IP of the pod.
 	IP string
+	// Status of the pod sandbox
```

Comment:
nit: Add a comment explaining that not every runtime uses this yet, so it could be nil (not set).